turbinia/workers/binary_extractor.py

```diff
@@ -41,21 +43,44 @@ def run(self, evidence, result):
     binary_extraction_evidence = BinaryExtraction()
 
     binary_extraction_evidence.local_path = self.output_dir
-    image_export_log = os.path.join(
-        self.output_dir, '{0:s}.log'.format(self.id))
+    image_export_log = os.path.join(self.output_dir, 'binary_extraction.log')
+
+    binary_extraction_dir = os.path.join(self.output_dir, 'extracted_binaries')
 
     cmd = [
-        'image_export.py', '--signatures', 'elf,exe_mz', '--logfile',
-        image_export_log
+        'image_export.py', '--partitions', 'all', '--no_vss', '--signatures',
+        'elf,exe_mz', '--logfile', image_export_log
     ]
     if config.DEBUG_TASKS:
       cmd.append('-d')
-    cmd.extend(['-w', self.output_dir, evidence.local_path])
+    cmd.extend(['-w', binary_extraction_dir, evidence.local_path])
 
     result.log('Running image_export as [{0:s}]'.format(' '.join(cmd)))
 
     self.execute(
         cmd, result, log_files=[image_export_log],
         new_evidence=[binary_extraction_evidence], close=True)
 
+    json_path = os.path.join(
+        self.output_dir, 'extracted_binaries', 'hashes.json')
+
+    with open(json_path) as json_file:
+      hashes = json.load(json_file)
+
+    files = next(os.walk(binary_extraction_dir))[2]
+    hash_cnt = len(hashes)
+    binary_cnt = len(files) - 1
+
+    result.status = 'Extracted {0:d} hashes and {1:d} binaries from the ' \
+                    'evidence.'.format(hash_cnt, binary_cnt)
```

Comment:
You can also put parens around the string so that you don't have to escape the newline:

```
    result.status = ('Extracted {0:d} hashes and {1:d} binaries from the '
                     'evidence.'.format(hash_cnt, binary_cnt))
```

Reply:
Thanks for the suggestion, I changed that.